tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/executor/TestExecutor.java

```diff
@@ -303,5 +303,56 @@ public void test05() throws TornadoExecutionPlanException {
         }
     }
 
+    /**
+     * Test Multi-Graphs in an execution plan. An execution plan can hold and launch
+     * multiple immutable task graphs. This tests shows how to execute individual immutable
+     * task graphs in any order.
+     * 
+     * @throws TornadoExecutionPlanException
+     */
+    @Test
+    public void test06() throws TornadoExecutionPlanException {
+        int numElements = 16;
+        IntArray a = new IntArray(numElements);
+        IntArray b = new IntArray(numElements);
+        IntArray c = new IntArray(numElements);
+
+        a.init(1);
+        b.init(2);
```

Comment:
can we please test the outcome when the elements of the arrays are initialised with different values?